datamodel/initdb.d/09_metadata.sql

```diff
@@ -0,0 +1,14 @@
+-- Assumes the PSQL client
+\set ON_ERROR_STOP true
+\connect dcsa_openapi
+
+-- Currently an implementation detail for the reference implementation but one
+-- we would like to remove eventually.
+BEGIN;
+
+-- Metadata for Booking table to avoid having to query shipmentEvent for date_times necessary for BookingResponseTO
+ALTER TABLE dcsa_im_v3_0.booking
+    ADD COLUMN IF NOT EXISTS event_created_date_time timestamp with time zone NOT NULL;
```

Comment:
I don't think we should call it `event_`created_date_time. Can we drop that part?

Reply:
@HenrikHL Should I rename it to `updated_date_time` and `created_date_time`?